pkg/apis/extensions/v1beta1/conversion.go

```diff
@@ -212,3 +213,7 @@ func Convert_networking_IngressSpec_To_v1beta1_IngressSpec(in *networking.Ingres
 	}
 	return nil
 }
+
```

Comment:
I had to manually add this conversion since it cannot automatically generate the conversion to v1beta1 since it does not have the newly added LastUpdateTime and shows a following warning in [pkg/apis/extensions/v1beta1/zz_generated.conversion.go](https://github.com/kubernetes/kubernetes/pull/108863/files#diff-e6b31c45b31b2f076d8c02296e4833e421faafea5258b7e33d86507dd989fb7d)

```
// WARNING: in.LastUpdateTime requires manual conversion: does not exist in peer-type
```

Is this okay or should I take some other approach?